database/index.js

```diff
@@ -29,36 +44,27 @@ const getReviewsByProductId = function (productid, callback) {
   });
 };
 
-// create review for one product
-const postReviewsByProductId = function (product, callback) {
-  const queryString = 'Insert into reviews (product_id, title, text, date, author, overall_rating, value_rating, quality_rating, appearance_rating, ease_of_assembly_rating, works_as_expected_rating, recommended, helpful_count, not_helpful_count) VALUES(?, ?, ?, ?,?, ?, ?, ?,?, ?, ?, ?,?, ?)';
-
-  connection.query(queryString, (Object.values(product)), (error, results, fields) => {
+// update
+const incrementReviewCounts = function (req, callback) {
```

Comment:
Does this update the number of reviews in the db? Is it used for your put request?

Reply:
Ah nevermind, I see it adds 1 to a number in a column